spec/sample_output/replica3_arbiter.vol

```diff
@@ -0,0 +1,79 @@
+volume vol1-client-0
+    type protocol/client
+    option transport.socket.read-fail-log false
+    option volfile-key /vol1
+    option remote-subvolume /exports/vol1/s1
+    option remote-host server1
+    option remote-port 4501
+end-volume
+
+volume vol1-client-1
+    type protocol/client
+    option transport.socket.read-fail-log false
+    option volfile-key /vol1
+    option remote-subvolume /exports/vol1/s2
+    option remote-host server2
+    option remote-port 4502
+end-volume
+
+volume vol1-client-2
+    type protocol/client
+    option transport.socket.read-fail-log false
+    option volfile-key /vol1
+    option remote-subvolume /exports/vol1/s3
+    option remote-host server3
+    option remote-port 4503
+end-volume
+
+volume vol1-replicate-0
+    type cluster/replicate
+    option data-self-heal on
+    option granular-entry-heal on
+    option iam-self-heal-daemon off
+    option metadata-self-heal on
+    option entry-self-heal on
+    option read-hash-mode 5
+    option arbiter-count 1
```

Comment:
In template, this is added before `data-self-heal` option

Reply:
Moved arbiter-count abover data-self-heal option in sample_output